content/en/service_owners_guide/run.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Maintaining and running your Datadog Implementation
+description: Setup and implement Datadog as a Service owner to avoid pit-falls down the road
+further_reading:
+- link: "getting_started/application/"
+  tag: "Documentation"
+  text: "Getting Started with Datadog"
+---
+
+In the previous two sections, we viewed Datadog through the lens of the plan and build phases, next let's look at the run phase. There are a series of tasks, both internal and external, that will keep the Datadog service running efficiently that need to be managed. 
+
+## Service tasks
+
+By releasing your new individual Datadog services in a rational and sequential pattern, gaining confidence as the service develops, we can reduce the risks of the implementation, and increase adoption from our users. In this section we will list a particular sequence of service item releases that will optimize your Datadog service user's experience. It is not practical to discuss all the intricacies involved in these items, as the diversity of the IT architecture at this level is too extensive to be captured in a single document. 
+
+Here are a few highlights:
+
+### Onboarding a new infrastructure instance
+
+Infrastructure is the foundational object of IT and observability.  It will be the first, and most common task of a Datadog service run team. The platform comes with incredible flexibility, and accelerators to complete the vast majority of this task, but all that capability still has to be molded into a shape that fits into your local conditions. HyperVisors, HyperScalers, Serverless Infrastructure, may all be part of your IT architecture.  
+
+**Recommendation**:   
+As mentioned in the ancillary products(link), it is highly recommended when you onboard new infrastructure to use [Fleet Automation][1] to aid in remotely managing your Datadog Agents at scale. Continuously monitor your teams for new infrastructure requests, flag them early and rationally apply engineering resources to focus on sensible expansions to infrastructure menu offerings.
```

Comment:
Recommending a beta product should come with a caveat I think

Reply:
good point! For the lifecycle product updates, Fleet is getting moved from Beta to NEW (GA) so that should help! thanks for the callout though :)